run.py

```diff
@@ -646,6 +646,10 @@ def run(self):
             check_value = self.check()
             if check_value["check_sh"]:
                 return check_value
+        
+        check_output = subprocess.call(["{}/check-eve.py".format(TOPDIR), outdir, "-q"])
+        if check_output != 0:
+            raise TestError("Invalid JSON schema")
 
         if not check_value["failure"] and not check_value["skipped"]:
             print("===> %s: OK%s" % (os.path.basename(self.directory), " (%dx)" % count if count > 1 else ""))
```

Comment:
The output was a little odd because this gets line gets printed before. So I moved the validation test to execute before that.